web/locales/pl.json

```
@@ -75,7 +75,7 @@
         "carbonintensity": "Emisja CO₂",
         "lowcarbon": "Niskoemisyjne źródła energii",
         "renewable": "Odnawialne źródła energii",
-        "crossborderexport": "Eksport",
+        "crossborderexport": "Eksport za granicę",
```

Tego bym nie zmieniał. Na mapie są też eksporty pomiędzy obszarami jednego kraju, np. w Norwegii.

```suggestion
        "crossborderexport": "Eksport",
```

---

Chyba że to będzie w sense "eksport za granicę obszaru". Ale nie wiem, mi się wydaje że "za granicę" nie jest tu bardzo potrzebne.